R/combine_lists.R

```diff
@@ -0,0 +1,9 @@
+# Helping function for overwriting elements of a base list with elements of a
+# second list; here we assume that both lists have names, and that the names of
+# new_list is a subset of the names of base_list
+combine_lists <- function(base_list, new_list) {
+    for (element_name in names(new_list)) {
+        base_list[[element_name]] <- new_list[[element_name]]
+    }
+    base_list
+}
```

Comment:
modifyList does this, but it would also add new elements. Not sure if you want that.

Reply:
I didn't know about `modifyList`. Whenever `combine_lists` is called, the two lists have already been checked to make sure that the new list only has elements that already in the base list. So that wouldn't be an important difference.

I will replace `combine_lists` with `modifyList`.